src/modules/claim/claim.controller.ts

```diff
@@ -269,7 +269,6 @@ export class ClaimController {
   public async getByUserDid(@Param('did') did: string, @User() user?: string) {
     return await this.claimService.getByUserDid({ did, currentUser: user });
   }
```

Comment:
```suggestion
  }
  
```

Reply:
@Harasz what is this change - adding a space?

Reply:
Yes, you removed a space here, idk why